layouts/shortcodes/release-binaries.html

```diff
@@ -0,0 +1,93 @@
+<!-- Fetch release_binaries.json from kubernetes-sigs/downloadkubernetes to render in table -->
+{{ $response := getJSON "https://raw.githubusercontent.com/dipesh-rawat/downloadkubernetes/integrate-k8s-website/dist/release_binaries.json" }}
+
+{{ $Binaries := slice }}
+{{ $AllOSes := slice }}
+{{ $AllArch := slice }}
+{{ $AllVersions := slice }}
+
+{{ range $key, $value := $response }}
+    {{ if eq $key "Binaries" }}
+        {{ $Binaries = $value }}
+    {{ else if eq $key "AllOSes" }}
+        {{ $AllOSes = $value }}
+    {{ else if eq $key "AllArch" }}
+        {{ $AllArch = $value }}
+    {{ else if eq $key "AllVersions" }}
+        {{ $AllVersions = $value }}
+    {{ end }}
+{{ end }}
+
+<link href="{{ "css/release_binaries.css" | relURL }}" rel="stylesheet">
+<script defer src="{{ "js/release_binaries.js" | relURL }}"></script>
+
+<div class="filter-options">
+    <div class="os-options">
+        <div>
+            <b>{{ T "release_binary_os_option" }}</b>
+            <div class="buttons" id="os-filter">
+                {{- range $AllOSes }}
+                <button class="button" data-os="{{.}}-data">{{.}}</button>
+                {{- end }}
+            </div>
+        </div>
+    </div>
+    <div class="version-options">
+        <div>
+            <b>{{ T "release_binary_version_option" }}</b>
+            <div class="buttons" id="version-filter">
+                {{- range $AllVersions }}
+                <button class="button" data-version="{{.}}-data">{{.}}</button>
+                {{- end }}
+            </div>
+        </div>
+    </div>
```

Comment:
We should show the current release (by current I mean “matching `Params.version`) only. If people want to download a different release we can make it clear how to select a different set of docs, and signpost them to the equivalent page for other available versions.

We can also signpost them to [downloadkubernetes.com](https://downloadkubernetes.com).

Reply:
Implemented the suggestion in the latest commit. Could you please review it and share your feedback?